src/Components/WebAssembly/Build/src/targets/Blazor.MonoRuntime.props

```diff
@@ -5,8 +5,7 @@
     <AdditionalMonoLinkerOptions>--disable-opt unreachablebodies --verbose --strip-security true --exclude-feature com -v false -c link -u link -b true</AdditionalMonoLinkerOptions>
 
     <_BlazorJsPath Condition="'$(_BlazorJsPath)' == ''">$(MSBuildThisFileDirectory)..\tools\blazor\blazor.webassembly.js</_BlazorJsPath>
-    <_BaseBlazorDistPath>dist\</_BaseBlazorDistPath>
```

Comment:
I had to account for these in path manipulation. Removed since this isn't used any more.